templates/copy-lambdas.template

```diff
@@ -152,7 +152,9 @@
                     "Version": "2012-10-17"
```

Comment:
Can we use the submodule here as well - https://github.com/aws-quickstart/lambda-copyzips instead of maintaining the same code here

See : https://github.com/aws-quickstart/quickstart-aws-acm-certificate/blob/main/templates/quickstart-aws-acm-certificate.template.yml for examples